hyperspace/metrics/src/handler.rs

```diff
@@ -82,6 +82,15 @@ impl MetricsHandler {
 
 	pub async fn handle_events(&mut self, events: &[IbcEvent]) -> anyhow::Result<()> {
 		for event in events {
+			let latest_processed_height = self.metrics.latest_processed_height.get();
+			// fn height() isn't defined on all IbcEvents
+			if matches!(event,
+				IbcEvent::SendPacket(_) |  IbcEvent::ReceivePacket(_) | 
+				IbcEvent::WriteAcknowledgement(_) | IbcEvent::AcknowledgePacket(_) | IbcEvent::TimeoutPacket(_) |
+				IbcEvent::TimeoutOnClosePacket(_) | IbcEvent::UpdateClient(_))
+			 &&  event.height().revision_height < latest_processed_height {
+				continue
+			}
```

Comment:
This probably should be `!matches((IbcEvent::X && IbcEvent::Y && ..) ||  event.height().revision_height < latest_processed_height)`